include/readelements.php

```diff
@@ -0,0 +1,541 @@
+<?php
```

Comment:
This file belongs in the modules/formulize/include folder.

Reply:
Actually, I see you have a copy of this with changes in the right place lower down. So this copy can just be removed from the branch.

Reply:
removed